src/ui/components/app/importers/DeleteImporterDialog.vue

```diff
@@ -1,41 +1,39 @@
 <template>
```

Comment:
I just removed the root `v-row`.

Before:
![image](https://user-images.githubusercontent.com/17686879/104820405-1eec8b00-583d-11eb-9117-c0feae8255d6.png)

After:
![image](https://user-images.githubusercontent.com/17686879/104820412-27dd5c80-583d-11eb-881d-ccf7adde953f.png)